Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/ExchangeManagerBean.java

```diff
@@ -26,29 +26,28 @@
  */
 package gov.hhs.fha.nhinc.admingui.managed;
 
+import static gov.hhs.fha.nhinc.admingui.util.HelperUtil.execPFHideDialog;
```

Comment:
When ExchangeManagemnet Webservices are not deployed, how the view and this bean will behave? Is it an exception or is user appropriately notified?

Reply:
the interface will still function but it wont do anything; do we have plan for what message to return?